.github/CODEOWNERS

```diff
@@ -126,3 +126,8 @@ android/android_browser_tests.gni @brave/android-tests-reviewers
 # have that owners record temporary to avoid people adding
 # new things there
 android/java/org/chromium/chrome/browser/preferences/BravePrefServiceBridge.java @AlexeyBarabash @samartnik @SergeyZhukovsky
+
+# GitHub configuration
+/.github/ @brave/devops-team
+/.github/CODEOWNERS @brave/devops-team @brave/sec-team @bsclifton
```

Comment:
I think it's fine to leave the sec team out of this one since:
- it wouldn't be blocking (either devops or clifton can also approve)
- if devops or clifton are unsure, they can of course always ping sec-team